force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -44,7 +47,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The difference between the two numbers.
      */
     public static Integer subtraction(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        integer result;
+        if( a !=null && b !=null){
```

Comment:
Try to be consistent with spacing when formatting your code. `if (a != null && b != null) {`